sklearn/cluster/hierarchical.py

```diff
@@ -694,6 +692,12 @@ def fit(self, X, y=None):
         -------
         self
         """
+        if self.pooling_func != 'deprecated':
+            warnings.warn('"pooling_func" as a parameter argument is '
```

Comment:
Actually, it surprises me that this has test coverage. Please find out if there is a use of `AgglomerativeClustering(pooling_func=...)` in the codebase and remove it.

Reply:
I don't think there's a use of `pooling_func`. there's no test covering pooling_func.